src/utils/validation.js

```diff
@@ -66,23 +66,23 @@ const getFieldName = (errors) => {
   return fieldName;
 };
 
-const stafferFormErrorField = (errors) => {
+const stafferOrCreateFormErrorField = (errors) => {
```

Comment:
nit: Instead of staffer, I would say to pick a name that can depict use-cases for Org field on both course create and staff create form (e.g. OrganizationFormFieldFailError)

Reply:
this is a function used by `handleStafferOrCreateFormError` below. That is called when a form submission fails (by redux form). The errors param here contains info about all the errors on the relevant form (staffer or create) and returns a field because of which the form submission has failed. Since it is not particular to the organization field, I feel the name should stay as is